pkgs/development/libraries/fontconfig/default.nix

```diff
@@ -68,7 +68,6 @@ stdenv.mkDerivation rec {
     cd "$out/etc/fonts"
     rm conf.d/{50-user,51-local}.conf
```

Comment:
I know you didn't change this part, but why is this removed? Why are all font configurations in `configuration.nix` or wherever not converted to XML in `/etc/fonts/local.conf` as is the default?

I think a lot the problems we are having with priorities are due to this. And now that our fontconfig setup is different from other major OSes, the documentation of fontconfig configuration on e.g. the arch wiki or elsewhere on the internet does not apply to NixOS.